src/GMXAutomationBase.sol

```diff
@@ -75,53 +75,66 @@ contract GMXAutomationBase is Ownable2Step {
     // INTERNAL FUNCTIONS
     ///////////////////////////
 
-    /// @notice Pushes the feedIds for marketProps: indexToken, longToken and shortToken to the feedIdToMarketTokenMap
+    /// @notice Pushes the feedIds for marketProps: indexToken, longToken and shortToken to the marketTokenToFeedId
     /// @dev Does not allow for duplicate feedIds or zero address feedIds
     /// @dev Does not push the Props.marketToken feedId to the feedIdToMarketTokenMap
     /// @param marketProps the Market Props struct to retrieve the feedIds from
-    function _addPropsToMapping(Market.Props memory marketProps) internal {
-        if (marketProps.indexToken != address(0)) {
-            uint256 indexTokenFeedId = uint256(i_dataStore.getBytes32(Keys.realtimeFeedIdKey(marketProps.indexToken)));
-            if (indexTokenFeedId == 0) revert GMXAutomationBase_ZeroIndexTokenFeedId();
-            if (!s_feedIdToMarketTokenMap.contains(indexTokenFeedId)) {
-                s_feedIdToMarketTokenMap.set(indexTokenFeedId, marketProps.indexToken);
-            }
+function _addPropsToMapping(Market.Props memory marketProps) internal {
```

Comment:
Looks like some formatting issues? If you run `forge fmt` in the command line, this should resolve.